packages/wallet/frontend/src/lib/api/account.ts

```diff
@@ -13,8 +13,7 @@ export const fundAccountSchema = z.object({
     .number({
       invalid_type_error: 'Please enter a valid amount'
     })
-    .min(1, { message: 'Please enter an amount' })
-    .positive()
+    .nonnegative({ message: 'Please enter an amount' })
```

Comment:
My fault. It should be `positive` instead of `nonnegative`. `nonnegative` allows number that are >= 0.
```suggestion
    .positive({ message: 'Please enter an amount' })
```